client/landing/gutenboarding/components/header/style.scss

```diff
@@ -27,6 +27,9 @@
 
 .gutenboarding__header-wp-icon {
 	color: var( --studio-blue-90 );
+	height: 24px;
+	width: 24px;
```

Comment:
It looks like @christos is handling this size change will be handled in #40176.

The approach there is preferable:

https://github.com/Automattic/wp-calypso/blob/024a291706f8d6bcc0a074f539a97d11497fb49b/client/landing/gutenboarding/components/header/index.tsx#L218

Maybe roll this back and let that PR handle it?